api/package.json

```diff
@@ -16,6 +16,7 @@
     "ejs": "^3.1.6",
     "express": "~4.16.1",
     "express-session": "^1.17.1",
+    "express-validator": "^6.12.0",
```

Comment:
I don't think we ended up using this; can we remove it?

Reply:
fixed!